build.sbt

```diff
@@ -29,7 +29,7 @@ lazy val appDependencies: Seq[ModuleID] = compile ++ test
 lazy val compile = Seq(
   ws,
   "uk.gov.hmrc" %% "fraud-prevention" % "0.10.0",
-  "uk.gov.hmrc" %% "microservice-bootstrap" % "10.6.0",
+  "uk.gov.hmrc" %% "bootstrap-play-25" % "4.13.0",
   "uk.gov.hmrc" %% "domain" % "5.6.0-play-25",
   "uk.gov.hmrc" %% "auth-client" % "2.23.0-play-25",
```

Comment:
This one can be upgraded to 2.26.0-play-25 (https://catalogue.tax.service.gov.uk/service/api-platform-test)

Reply:
Done